pkgs/a-h/templ/pkg.yaml

```diff
@@ -1,8 +1,10 @@
 packages:
   - name: a-h/templ@v0.3.833
   - name: a-h/templ
-    version: v0.0.153
+    version: v0.2.639
```

Comment:
This test data doesn't match with the version constraint.

```yaml
- version_constraint: semver("< 0.2.639")
```

Reply:
Oops! Mistake when manually bisecting first working version. Fixed in bb1a822a33eb817788356317c6db1b8667d71f73